deploy/ray-operator/controllers/raycluster_controller.go

```diff
@@ -29,45 +28,14 @@ import (
 
 var log = logf.Log.WithName("RayCluster-Controller")
 
-// Add creates a new RayCluster Controller and adds it to the Manager with default RBAC. The Manager will set fields on the Controller
-// and start it when the Manager Started.
-func Add(mgr manager.Manager) error {
```

Comment:
The code is never executed, thus we did not watch the pod resource successfully.